src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/RequesterMapper.java

```diff
@@ -1,14 +1,27 @@
 package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
 
+import java.util.Optional;
 import org.hl7.fhir.dstu3.model.Practitioner;
 import org.springframework.stereotype.Component;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
-
-import java.util.Optional;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.RequesterNameAndAddress;
 
 @Component
 public class RequesterMapper {
```

Comment:
I wonder if these should actually be named for the return type of the `map` method. They all accept a `message`, but return different (possibly `Optional`) things.
```suggestion
public class PractitionerMapper {
```

Reply:
@bartek-sarul has pointed out that a fhir `Practitioner` could be several things from an edifact message, so if we make this kind of change then the responsibility of each mapper is to do the mappings _to_ the named fhir resource, and may expose multiple methods to do that according to need:

```
class PractitionerMapper {
    public Practitioner mapRequester(Message message) { ... }
    public Practitioner mapPerfomer(Message message) { ... }
}
```

Reply:
I think at the time of creating these, we were following the naming conventions in the examples on the ticket, which were inline with the message spec mapping diagram.  

Personally, i like to name methods based on their return type, as you suggest.

Reply:
Agreed with @matthewdoreilly that I'll rename this mapper in this PR and the others will be renamed in his - on the basis of keeping PRs clean and relevant to the respective stories.